protos/cloudevent.proto

```diff
@@ -21,12 +21,13 @@ message CloudEvent {
   // Optional & Extension Attributes
   map<string, CloudEventAttributeValue> attributes = 5;
   map<string, string> metadata = 6;
+  string datacontenttype = 7; // MIME type
```

Comment:
We cannot add this as it would mean deviating from the official schema https://github.com/cloudevents/spec/blob/main/cloudevents/formats/cloudevents.proto

Reply:
removed